controllers/users.js

```diff
@@ -563,6 +635,186 @@ class Users {
       }
     });
   }
+
+  /**
+   *
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} get all users
+   */
+  static async getAllUsers(req, res) {
```

Comment:
@shemaeric I don't see any difference between this controller and the one named `getAllAuthors` on line 337. They all return a list of users we have in the DB although the naming is different.